### PR TITLE
[com_fields] - fix show on side/admin/both

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #23246 .

### Summary of Changes
Checked if the field should be showed on the form

### Testing Instructions

create edit an article custom field 
set show on to **adminstrator** only

### Expected result
the field is not showed on from **site**

### Actual result
the field is showed on from **site** despite settings

### CODE REVIEW

File: administrator/components/com_fields/libraries/fieldsplugin.php [hunk 1]
```diff
@@ -116,6 +116,12 @@ public function onCustomFieldsPrepareField($context, $item, $field)
 		{
 			return;
 		}
+
```
Comment (Quy) [inline administrator/components/com_fields/libraries/fieldsplugin.php]: Remove tab

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 322f2c21bda3d920c5f5be45fc57958f6bd3f42c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24397">issues.joomla.org/tracker/joomla-cms/24397</a>.</sub>

**Comment by ghost** [comment]
@boomsya please test.

**Comment by HLeithner** [comment]
Thx, I tested it my self.

**Comment by duvi** [comment]
Actually, this has broken field display.

I think the "Show on" setting should only affect where the field should be visible in the **edit view** - in admin, in front end or both.

However, now the field is not displayed any more in the **article view** if the "Show on" setting is set to "Administrator", regardless of the "Automatic display" setting under "Render options".

**Pull request merged**